components/automate-ui/src/app/pages/+compliance/+reporting/reporting-searchbar/reporting-searchbar.component.ts

```diff
@@ -14,6 +14,9 @@ import {
   debounceTime, switchMap, distinctUntilChanged
 } from 'rxjs/operators';
 import { FilterC } from '../types';
+import {
+  ReportQueryService
+} from '../../shared/reporting';
```

Comment:
Minor point: our loose convention is to avoid `..` in pathing of imports. Things should either start with `app/...` or `./` (single dot for stuff in the current directory).

Reply:
Agreed, Updated!